spec/support/landing_folder_support/file_objects/et3_pdf_file/base.rb

```diff
@@ -1,82 +1,13 @@
 require 'rspec/matchers'
 require 'pdf-forms'
-require_relative '../base_pdf_file'
+require_relative '../base_pdf_section'
 module EtApi
   module Test
     module FileObjects
       # Represents the ET3 PDF file and provides assistance in validating its contents
       module Et3PdfFileSection
-        class Base < ::EtApi::Test::FileObjects::BasePdfFile
-          UndefinedField = Object.new
+        class Base < ::EtApi::Test::FileObjects::BasePdfSection
```

Comment:
Sections and files are now different as sections dont read the pdf file anymore - they expect it pre parsed